blockchain/blockchain.go

```diff
@@ -181,6 +181,7 @@ func (b *Blockchain) GetAvgGasPrice() *big.Int {
 func NewBlockchain(
 	logger hclog.Logger,
 	dataDir string,
+	levelOpt *leveldb.Options,
```

Comment:
Could we wrap it in `config`?

Reply:
`config` is `*chain.Chain` type, storage configuration should not be placed inside the `chain.Chain` configuration

Reply:
Alright, you convinced me.